extensions/wikia/EditPageLayout/css/core/wide.scss

```diff
@@ -263,6 +241,35 @@
 	}
 }
 
+#EditPage {
+	&.toolbar-expanded {
+		.preview_box {
+			border: 0;
+			clear: both;
+
+			h3 {
+				margin-top: 8px;
+			}
+
+			.preview_mobile {
+				padding-top: 3px;
+			}
+
+			.preview_desktop {
+				padding-top: 5px;
+			}
+		}
+
+		#wpDiff.diff-link {
```

Comment:
It looks like you're overriding some css selectors here. Would be great if we don't have to load style definitions we don't use.

As well per coding conventions it's advised to define css selectors rather with classes than ids.